packages/demo/src/index.js

```diff
@@ -39,12 +39,31 @@ async function main() {
   });
 
   // Uncomment out this if you want to demo the player package
-  // const player = webplayer(root);
+  //const player = webplayer(root);
 
-  function load() {
-    player.load(manifestInput.value);
+  async function load() {
+    await player.load(manifestInput.value);
+    populateQualityPicker();
   }
 
+  function populateQualityPicker() {
+    // Reset/Clear-out the drop down menu.
+    for (var i = qualityPicker.options.length - 1; i >= 0; i--) {
+      if (qualityPicker.options[i].value != '-1') qualityPicker.remove(i);
+    }
+
+    const videoQualities = player.getAllVideoQualities();
+    videoQualities.forEach((element, level) => {
```

Comment:
The IVideoQuality type should have the `id` as part of the object instead, there is no guarantee that the Ids are numbers for different types of playback engines.

Reply:
Ah. Okej 👍